store/main.go

```diff
@@ -53,10 +53,15 @@ func AddEmbedding(
 	ctx context.Context,
 	userId string,
 	filePath string,
-	text string,
+	chunkText string, //renamed from text
+	embeddingText *string, //using nil for default behavior
```

Comment:
Nice, just missing a spacing in the comments:
```suggestion
	chunkText string, // renamed from text
	embeddingText *string, // using nil for default behavior
```

Reply:
Fixed the spacing.